### PR TITLE
Feature/sc 53

### PR DESCRIPTION
**Pull request by mclaurentiu**

### CODE REVIEW

File: resources/ingestdocs.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# 27.12.2020 - mclaurentiu79@gmail.com
```
Comment (costezki) [inline resources/ingestdocs.sh:3]: very nice tool to have.

File: resources/ingestmeta.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# 27.12.2020 - mclaurentiu79@gmail.com
```
Comment (costezki) [inline resources/ingestmeta.sh:3]: nice tool

File: resources/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+elasticsearch
```
Comment (costezki) [inline resources/requirements.txt:1]: we can add `rdflib`

**Review by costezki — APPROVED** [review]
well done!

File: adapters/cellar_adapter.py [hunk 4]
```diff
@@ -30,155 +28,266 @@ def __init__(self):
         self.run = '+Run+Query+'
         self.max_query_size = 8000
 
-    def get_treaties(self, limit: int = None) -> dict:
-        """
-        Method to retrieve works of treaties and their metadata
-        :param limit: limit of query results
-        :return: dict with metadata
-        """
+    def _make_request(self, query):
```
[the remark this answers is not in the document]
Comment (mclaurentiu) [reply adapters/cellar_adapter.py:31]: @cosletmihai can you answer this please?

File: adapters/cellar_adapter.py [hunk 5]
```diff
@@ -30,155 +28,266 @@ def __init__(self):
...
+                                ?manif_html cdm:manifestation_type ?type_html.
+                                FILTER(str(?type_html) in ('html', 'xhtml'))}}
+                                BIND(IF(BOUND(?title),?title,'The title does not exist in that language'@en) as ?title_)
+                                BIND(IRI(concat(?manif_pdf,"/zip")) as ?pdf_to_download)
+                                BIND(IRI(concat(?manif_html,"/zip")) as ?html_to_download)
+                    }
+                    
+                    order by ?date_document"""
 
         response = self._make_request(query)
```
[the remark this answers is not in the document]
Comment (mclaurentiu) [reply adapters/cellar_adapter.py:127]: @cosletmihai Can you answer this please ?

File: adapters/cellar_adapter.py [hunk 6]
```diff
@@ -30,155 +28,266 @@ def __init__(self):
...
+        current_item = 0
+
+        for item in covid19_items['results']['bindings']:
+            current_item += 1
+            filename = hashlib.sha256(item['title_']['value'].encode('utf-8')).hexdigest()
+
+            filename_pdf = filename + '_pdf.zip'
+            filename_html = filename + '_html.zip'
+
+            try:
```
[the remark this answers is not in the document]
Comment (mclaurentiu) [reply adapters/cellar_adapter.py:143]: How do you know which one *really* exists ?

**Pull request merged**